src/app/pages/home/home.jsx

```diff
@@ -0,0 +1,123 @@
+
+/**
+ * Dependencies.
+ */
+
+var React = require('react');
+var StoreMixin = require('fluxible').StoreMixin;
+var GoogleMapsStore = require('../../maps/google-maps-store');
+var NavLink = require('../../nav/nav-link');
+var loadGoogleMaps = require('../../maps/load-google-maps');
+
+/**
+ * Home page.
+ */
+
+module.exports = React.createClass({
+  mixins: [StoreMixin],
+  statics: {
+    storeListeners: [GoogleMapsStore]
+  },
+  getInitialState: function () {
+    var venues = this.props.context.getStore('venue').getVenues();
+
+    return {
+      venues: venues
+    }
+  },
+  componentDidMount: function () {
+    var mapsIsReady = this.props.context.getStore('google-maps').isReady();
+    if (mapsIsReady) {
+      this.renderMap();
+    } else {
+      loadGoogleMaps(this.props.context);
+    }
+  },
+  render: function() {
+    return (
+      <div id="home-page">
+        <div id="map-canvas" ref="mapCanvas"></div>
```

Comment:
It defines an internal named reference so that the virtual dom element may be accessed at a later time. This is the way you can get a real dom reference so that you can integrate with 3rd party stuff (like google maps).